packages/tools/devtools/devtools-example/src/ClientUtilities.ts

```diff
@@ -4,105 +4,104 @@
  */
 
 import { ConnectionState } from "@fluidframework/container-loader";
-import { ContainerSchema, IFluidContainer } from "@fluidframework/fluid-static";
+import { IContainer } from "@fluidframework/container-definitions";
+import { ITelemetryBaseLogger } from "@fluidframework/tinylicious-client";
+import { SessionStorageModelLoader, StaticCodeLoader } from "@fluid-example/example-utils";
+import { CollaborativeDevtoolExampleAppObjects } from "./FluidObject";
 import {
-	ITelemetryBaseLogger,
-	TinyliciousClient,
-	TinyliciousContainerServices,
-} from "@fluidframework/tinylicious-client";
+	CollaborativeDevtoolExampleObjectsContainerRuntimeFactory,
+	ICollaborativeDevtoolExampleAppModel,
+} from "./Container";
 
 /**
  * This module contains Fluid Client utilities, including Container creation / loading.
  */
 
-/**
- * Type returned from when creating / loading the Container.
- */
-export interface ContainerLoadResult {
-	container: IFluidContainer;
-	services: TinyliciousContainerServices;
-}
-
 /**
  * Basic information about the container, as well as the associated audience.
  */
 export interface ContainerInfo {
 	/**
-	 * The initialized Fluid Container.
+	 * The initialized Container.
 	 */
-	container: IFluidContainer;
+	container: IContainer;
 
 	/**
 	 * The Container's unique ID. Also referred to as the "Document ID".
 	 */
 	containerId: string;
+
+	/**
+	 * Model object for the Container.
+	 */
+	obj: CollaborativeDevtoolExampleAppObjects;
 }
 
-function initializeTinyliciousClient(logger?: ITelemetryBaseLogger): TinyliciousClient {
-	console.log(`Initializing Tinylicious client...`);
-	return new TinyliciousClient({
+/**
+ * Creates a new Fluid Container from the provided client and container schema.
+ */
+export function createLoader(
+	logger?: ITelemetryBaseLogger,
+): SessionStorageModelLoader<ICollaborativeDevtoolExampleAppModel> {
+	const codeLoader = new StaticCodeLoader(
+		new CollaborativeDevtoolExampleObjectsContainerRuntimeFactory(),
+	);
+	const loader = new SessionStorageModelLoader<ICollaborativeDevtoolExampleAppModel>(
+		codeLoader,
 		logger,
-	});
+	);
+
+	return loader;
+}
+
+/**
+ * Creates a new Fluid Container from the provided client and container schema.
+ */
+async function createContainer(
+	loader: SessionStorageModelLoader<ICollaborativeDevtoolExampleAppModel>,
+): Promise<ContainerInfo> {
+	const createResponse = await loader.createDetached("1.0");
```

Comment:
The old code did some console logging here to note each step in the creation / loading process. Could we re-add that to this new create/load code?

Reply:
Adding logs for container creation and re-load.